### PR TITLE
docs: add comments to /build and /cmd files

### PR DESCRIPTION
**Pull request by geoffturk**
Part of #459

### CODE REVIEW

File: cmd/validation/main.go [hunk 1]
```diff
@@ -6,14 +11,25 @@ import (
 )
 
 func main() {
```
Comment (maxsarratt) [inline cmd/validation/main.go:13]: Do we need to add a lot of comments? It's good; it's self-explanatory.

Comment (maxsarratt) [reply cmd/validation/main.go:13]: @TingSyuanWang Not sure about your idea, but I think the comments make this simple code harder to understand.

**Review by maxsarratt — CHANGES_REQUESTED** [review]
A majority of the comments are not required. It would be more effective if we updated the README for each service to describe their objectives, instead of commenting throughout the code.

File: cmd/validation/main.go (hunk 1 above)
Comment (TingSyuanWang) [reply cmd/validation/main.go:13]: @maxsarratt Adding excessive comments to parts of the code that do not require special explanations can be unnecessary. The code itself may already be self-explanatory and easily understood. Thus, I concur with your viewpoint that it is advisable to refrain from adding comments in such cases.

**Comment by geoffturk** [comment]
> A majority of the comments are not required. It would be more effective if we updated the README for each service to describe their objectives, instead of commenting throughout the code.

OK, point well-taken. I'll scrap this PR and create another one with the detailed info in a README rather than injected directly in the code.

**Pull request closed without merging**